src/sfptpd_clock.c

```diff
@@ -543,7 +562,8 @@ static void fixup_clock(struct sfptpd_clock *clock, struct sfptpd_config_general
         /* Now that we have configured the clock's readonly flag, we can finally load saved frequency corrections
            and epoch startup correction.
         */
-        sfptpd_clock_correct_new(clock);
+	if (!cfg->clocks.no_initial_correction)
```

Comment:
I _wondered_ what this flag was for, when I saw it being initialised in the config.  Seems like a useful flag to have, is it deliberately not being documented?

Reply:
This flag is designed for `tstool` where it would default to _on_ and not exposed to the `sfptpd` front end - do you really want to support this as an option to _sfptpd_ proper? Has it been requested? Would you ever recommend it?